src/app/features/home/capture-tab/capture-item/capture-item.component.ts

```diff
@@ -72,8 +72,7 @@ export class CaptureItemComponent {
   );
 
   readonly hasCaption$ = this.proof$.pipe(
-    switchMap(proof => this.diaBackendAssetRepository.fetchByProof$(proof)),
-    map(asset => asset.caption !== '')
+    map(proof => proof.caption.trim() !== '')
```

Comment:
Is there any reason why `trim()` is added?

Reply:
@shc261392, just for safety. But I believe backend handles empty strings therefore I removed trim() in this commit [refactor(capture-item): remove unnecessary .trim() on caption](https://github.com/numbersprotocol/capture-lite/pull/3157/commits/c4a0040a4852736de2543111c1f555eff2f4b9b3)